raiden/storage/migrations/v19_to_v20.py

```diff
@@ -175,10 +189,10 @@ def _add_onchain_locksroot_to_snapshot(
                 channel["our_state"]["onchain_locksroot"] = serialize_bytes(our_locksroot)
                 channel["partner_state"]["onchain_locksroot"] = serialize_bytes(partner_locksroot)
 
-    return json.dumps(snapshot, indent=4), snapshot_record.identifier
+    return json.dumps(snapshot, indent=4), snapshot_record.state_change_identifier
```

Comment:
Not sure if this is what we want here: `snapshot_record.identifier` should/does exist and is the unique id for the affected snapshot. Why did you change it?

Reply:
Because `snapshot_record` is a `StateChangeRecord` (at least according to the function annotation) and therefore mypy (and PyCharm) complained.

Reply:
yeah, the typing of the signature is incorrect then: https://github.com/raiden-network/raiden/pull/4074/files#diff-30896e13fdd21557ed5512f90093e729R196

Reply:
Ok makes sense. I changed it.